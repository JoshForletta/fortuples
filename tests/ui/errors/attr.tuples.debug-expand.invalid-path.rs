use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::debug_expand(path = 2)]
    impl Container<#Tuple> {}
}

fn main() {}
