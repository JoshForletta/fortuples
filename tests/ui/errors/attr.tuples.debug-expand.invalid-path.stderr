error: a filepath is expected
 --> tests/ui/errors/attr.tuples.debug-expand.invalid-path.rs:6:35
  |
6 |     #[tuples::debug_expand(path = 2)]
  |                                   ^
